src/map/entities/mobentity.cpp

```diff
@@ -603,7 +603,14 @@ void CMobEntity::Spawn()
         }
     }
     
-    m_DespawnTimer = time_point::min();
```

Comment:
adding on to the second part... why use bigmobmod instead of just using std::chrono::seconds?